src/client/datascience/jupyter/jupyterSession.ts

```diff
@@ -133,6 +134,9 @@ export class JupyterSession extends BaseJupyterSession {
 
             // Make sure it is idle before we return
             await this.waitForIdleOnSession(newSession, this.idleTimeout);
+
+            // So that we don't have problems with ipywidgets, always register the default ipywidgets comm target.
+            newSession.kernel?.registerCommTarget(Identifiers.DefaultCommTarget, noop);
```

Comment:
This is what we have in raw kernels as well, we need to register comm target for `jupyter.widget`.